horace_core/sqw/axes_blocks/@AxesBlockBase/AxesBlockBase.m

```diff
@@ -367,6 +367,31 @@
             end
 
         end
+
+        function steps = get_bin_step(obj)
+            % Return the step in each dimension in units of img_range units
+            % Inputs:
+            % obj   -- initialized instance of an AxesBlockBase class
+            steps = (obj.img_range(2,:) - obj.img_range(1,:)) ./ obj.nbins_all_dims;
+        end
```

Comment:
I believe, step should be 
```
steps = (max_val-min_val)/(nbins-1)
```
fitst point is `min_val`, last point at nbins is: 
`x_nbins = min_val + step*(nbins-1) == max_val`

Reply:
As this will lead to inf for nbins == 0, steps should be along the projection axes only i.e. `obj.img_range(2,obj.pax)`